openrtb_ext/floors_test.go

```diff
@@ -0,0 +1,221 @@
+package openrtb_ext
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func getFlag(in bool) *bool {
+	return &in
+}
+
+func TestPriceFloorRulesGetEnforcePBS(t *testing.T) {
+
```

Comment:
Nitpick: Can you remove this blank line?

Reply:
Done